### DISCUSSION TITLE
Weird visual bug on the label of input

### DISCUSSION THREAD

**Discussion by danielrdrigues** [mdbootstrap/TW-Elements#1988] [Support from community]
After installing and testing an input from quickstart I'm getting this label getting wrong padding top and this vertical bars.
![image](https://github.com/mdbootstrap/Tailwind-Elements/assets/10256444/171c6bae-8a0a-4832-aa23-cac5686349d9)

This vertical bars are chrome exclusive, on firefox doesn't exist, just the top padding on firefox:
![image](https://github.com/mdbootstrap/Tailwind-Elements/assets/10256444/bbf6eb63-23e4-4b83-934d-8b3820338dc1)

**Comment by juujisai** [comment]
Hi! Can you show the code for input you used in your app? Does the bars show for you on some specific window width? Is this a fresh install of vanilla JS app?
